src/main/java/com/github/devotedmc/hiddenore/listeners/WorldGenerationListener.java

```diff
@@ -65,42 +60,42 @@ public WorldGenerationListener(ConfigurationSection config) {
 			replaceWith = Material.matchMaterial(with);
 		}
 	}
-	
+
 	/**
 	 * Reviews the chunk line by line and replaces all instances of toReplace with replaceWith.
 	 * This is configured world to world.
-	 * 
+	 *
 	 * Note that by contract, this is called for a single chunk but the generation can occur
 	 * for surrounding chunks, if they are not yet populated.
-	 * 
-	 * @param event ChunkPopulateEvent covering the chunk 
+	 *
+	 * @param event ChunkPopulateEvent covering the chunk
 	 */
 	@EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
 	public void postGenerationOreClear(ChunkPopulateEvent event) {
 		if (toReplace == null || replaceWith == null || worldName == null) {
 			return;
 		}
-		
+
 		Chunk chunk = event.getChunk();
-		
+
 		World world = chunk.getWorld();
-		
+
```

Comment:
Line after this is where we do a last check that the world triggering this event is the same as what this listener is listening for. Also a few lines up (at start of method) we failfast if the local worldName is null.

Def needs setting.